deis-workflow-manager/README.md

```diff
@@ -0,0 +1,7 @@
+# deis-workflow-manager
+
+Describe your chart here. Link to upstream repositories, Docker images or any
+external documentation.
+
+If your application requires any specific configuration like Secrets, you may
+include that information here.
```

Comment:
I think that might be the output of `helm create`.

https://github.com/helm/helm/blob/master/action/create.go